### ISSUE TITLE
Panorama Custom Width

### ISSUE THREAD

**Issue by MinecraftMoments** [AprilSylph/XKit-Rewritten#814]
### Concept

As someone who wants a wider space to read tumblr posts without needing to zoom in, a customizable-width option to the panorama (either by pixels or percentage) would be very helpful.

**Comment by Skaramuche** [comment]
agreed--I have a high-rez monitor, and definitely don't need tumblr to be full-width, which makes the images so tall they won't all fit onscreen at once; but making it twice as wide would be great.
